src/test/helpers.ts

```diff
@@ -106,7 +106,7 @@ export const testFramework = (
    * Create a helper function for running tests against the given serve handler.
    */
   const run = async (
-    handlerOpts: Parameters<typeof handler["serve"]>,
+    handlerOpts: Parameters<(typeof handler)["serve"]>,
```

Comment:
New Prettier version, I'm assuming

Reply:
Could pin `devDependencies`, but now I'm bikeshedding so ignore me

Reply:
@goodoldneon Oh, no, as in I explicitly had to do `yarn upgrade prettier` or Prettier can't even parse the file. 😅 Very intentional upgrade. :)